recipes/pylance/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "pylance" %}
+{% set version = "0.9.8" %}
+{% set build_number = environ.get('GIT_DESCRIBE_NUMBER', '0') %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lancedb/lance/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: b1152fd112214e2974280964ba934202e7e8b736cd768177d0ff25dc80663392
+
+build:
+  number: 0
+  # There's a protocol buffer related error...
+  # This recipe skips windows until it gets fixed...!
+  # https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=862176&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=7c0f8eae-6d6f-51bf-636f-73a1a7fb1bc4
+  skip: true  # [py<38 or win]
```

Comment:
Please remove the skip and let's have a look together at this.

Reply:
I've updated the file to remove the cross-compile related settings and the skip part for `win`.